swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -273,6 +285,44 @@ public final class Store: ObservableObject {
     try ipcClient().start(token: token)
   }
 
+  private func beginConfigurationPolling() {
+    // Ensure we're idempotent if called twice
+    if self.configurationTimer != nil {
+      return
+    }
+
+    let updateConfiguration: @Sendable (Timer) -> Void = { _ in
+      Task {
+        await MainActor.run {
+          self.configurationUpdateTask?.cancel()
+          self.configurationUpdateTask = Task {
+            if !Task.isCancelled {
+              do {
+                self.configuration = try await self.ipcClient().getConfiguration()
+              } catch {
+                Log.error(error)
+              }
+            }
+          }
+        }
+      }
+    }
+
+    let intervalInSeconds: TimeInterval = 1
+    let timer = Timer(timeInterval: intervalInSeconds, repeats: true, block: updateConfiguration)
+
+    RunLoop.main.add(timer, forMode: .common)
+    self.configurationTimer = timer
+    updateConfiguration(timer)
```

Comment:
Is this necessary? Seems like we would be called in at most 1s anyway, right?

Reply:
Yeah, probably true. I'll remove it.